src/app_service/service/network/service.nim

```diff
@@ -58,6 +58,13 @@ proc resetNetworks*(self: Service) =
 proc getCombinedNetworks*(self: Service): seq[CombinedNetworkDto] =
   return self.fetchNetworks()
 
+# TODO:: update the networks service to unify the model exposed from this service
```

Comment:
Is there already a task created for this TODO?

Reply:
https://github.com/status-im/status-desktop/issues/12717 created now